### PR TITLE
CTDC-1554

### PR DESCRIPTION
**Pull request by kuffelgr**
This PR updates the static content on the Data Submission page (https://clinical.datacommons.cancer.gov/#/submit).

### CODE REVIEW

File: aboutPagesContent.yaml [hunk 1]
```diff
@@ -2,7 +2,65 @@
...
+    - paragraph: " Those interested in submitting data to the CTDC must first fill out a data submission request through $$[the CRDC Submission Portal](https://hub.datacommons.cancer.gov/)$$. Requests will be reviewed to assess alignment with CTDC and CRDC’s mission and overall data requirements. Review times can be expected to be four to six weeks. For more information regarding submission requests and data submission, see $$[CRDC’s Submit Data page](https://datacommons.cancer.gov/submit)$$."
+    - paragraph: "Below are the primary categories of data the CTDC currently accepts:"
+    - paragraph: "De-identified Clinical Data:"
+    - listWithDots :
+      - "Patient demographics"
+      - "Clinical outcomes"
+      - "Treatment regimens"
+      - "Follow-up data (disease progrssion, etc.)"
```
Comment (jonkiky) [inline aboutPagesContent.yaml]: Should  "progrssion" be "progression." ?

File: aboutPagesContent.yaml [hunk 2]
```diff
@@ -2,7 +2,65 @@
...
+    - listWithDots :
+      - "Whole genome sequencing data"
+      - "Exome sequencing data"
+      - "Transcriptome data (e.g., RNA-Seq)"
+      - "Epigenetic data (e.g., methylation profiles)"
+      - "Variant and gene expression data"
+    - paragraph: "Biomarker Data:"
+    - listWithDots :
+      - "Proteomics"
+      - "Metablomics"
```
Comment (jonkiky) [inline aboutPagesContent.yaml]: Should "Metablomics" be "Metabolomics."?

File: aboutPagesContent.yaml [hunk 3]
```diff
@@ -2,7 +2,65 @@
   title: "Data Submission"
   primaryContentImage: https://raw.githubusercontent.com/CBIIT/datacommons-assets/ctdc_Assets/ctdc/images/aboutPages/About_CRDC.png
   content:
-    - paragraph: "CTDC is not accepting external data submissions at this time. Learn how to $$[submit data to other data repositories](type:internal url:https://datacommons.cancer.gov/data/submit-data target:_blank )$$ within the Cancer Research Data Commons."
+    - paragraph: "The Clinical and Translational Data Commons (CTDC) is designed to facilitate the integration and analysis of a wide array of clinical and translational research data. To support this initiative, the CTDC will accept a variety of data types, each crucial for the comprehensive understanding of clinical outcomes and the translational pathway from laboratory and clinical research to clinical applications.
```
Comment (Nahomtes) [inline aboutPagesContent.yaml]: Add a closing quotation mark at the end of the paragraph

File: aboutPagesContent.yaml (hunk 1 above)
Comment (kuffelgr) [reply aboutPagesContent.yaml]: Good catch.

File: aboutPagesContent.yaml (hunk 2 above)
Comment (kuffelgr) [reply aboutPagesContent.yaml]: Yes.

File: aboutPagesContent.yaml (hunk 3 above)
Comment (kuffelgr) [reply aboutPagesContent.yaml]: Eagle eye!

**Pull request merged**